operator/Makefile

```diff
@@ -122,8 +123,13 @@ deploy-lite: manifests
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
+# Commented out alternative is looking ahead to issue that on Openshift our v1 CRD is too large
+# to be installed. This may also affect operator-sdk community operators.
+# See https://github.com/operator-framework/operator-registry/issues/385
+# Solution may be to drop v1alpha2 and v1alpha3 versions to decrease size by 2/3
 manifests_v1: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd_v1/bases crd:crdVersions=v1
+	$(CONTROLLER_GEN) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd_v1/bases crd:crdVersions=v1
+	#$(CONTROLLER_GEN) rbac:roleName=manager-role webhook paths="./apis/machinelearning.seldon.io/v1" output:crd:artifacts:config=config/crd_v1/bases crd:crdVersions=v1
```

Comment:
Replicated comment

Reply:
They are not the same. This is the addition to possibly at later to only create v1 SeldonDeployment CRD